compat/src/forwardRef.js

```diff
@@ -27,16 +27,10 @@ export function forwardRef(fn) {
 	// We always have ref in props.ref, except for
 	// mobx-react. It will call this function directly
 	// and always pass ref as the second argument.
-	function Forwarded(props, ref) {
+	function Forwarded(props) {
```

Comment:
We might need to double check with `mobx-react` as outlined in the comment for this function if the second argument is still needed.

Reply:
Just checked, looks like this is not needed anymore 🎉  So the comment can be dropped too 👍